README.md

```diff
@@ -53,153 +46,113 @@ An OpenCL 1.2 compatible GPU with driver installed is required by the SDK. Follo
 
 A C++17 compiler is required.
 
-Ubuntu 20.04:
 ```bash
 sudo apt-get install -y g++
 ```
 
 ### Downloading and building Zivid ROS driver
 
-First, load the setup.bash script into the current session.
+First, source the `setup.bash` script for your ROS distribution in your terminal:
 
-Ubuntu 20.04:
 ```bash
-source /opt/ros/noetic/setup.bash
+source /opt/ros/$ROS_DISTRO/setup.bash
 ```
 
-Create the workspace and src directory:
+Then create the workspace and src directory:
 ```bash
-mkdir -p ~/catkin_ws/src
+mkdir -p ~/ros2_ws/src
 ```
 
 Clone the Zivid ROS project into the src directory:
 ```bash
-cd ~/catkin_ws/src
-git clone https://github.com/zivid/zivid-ros.git
+cd ~/ros2_ws/src
+git clone https://github.com/zivid/zivid-ros.git -b ros2-preview
```

Comment:
must remove the `-b ros2-preview`

Reply:
df24da9329e235dcb30399b53eb6c1781f73d634